types/mapbox__mapbox-gl-geocoder/index.d.ts

```diff
@@ -158,6 +158,30 @@ declare namespace MapboxGeocoder {
          * If `false`, indicates that the `localGeocoder` results should be combined with those from the Mapbox API with the `localGeocoder` results ranked higher. (optional, default false)
          */
         localGeocoderOnly?: boolean | undefined;
+        /**
+         * Specify whether to return autocomplete results or not. When autocomplete is enabled,
+         * results will be included that start with the requested string, rather than just responses
+         * that match it exactly. (optional, default true)
+         */
+        autocomplete?: boolean | undefined;
+        /**
+         *  Specify whether the Geocoding API should attempt approximate, as well as exact, matching
+         *  when performing searches, or whether it should opt out of this behavior and only attempt
+         *  exact matching. (optional, default true)
+         */
+        fuzzyMatch?: boolean | undefined;
+        /**
+         * Specify whether to request additional metadata about the recommended navigation
+         * destination corresponding to the feature or not. Only applicable for address features.
+         * (optional, default false)
+         */
+        routing?: boolean | undefined;
+        /**
+         * Filter results to geographic features whose characteristics are defined differently by
+         * audiences belonging to various regional, cultural, or political groups. (optional,
+         * default "us")
+         */
+        worldview?: string | undefined;
```

Comment:
Based on the [doc](https://docs.mapbox.com/api/search/geocoding/#available-worldviews) options for this field are `ar`, `cn`,`in`, `jp`, `ma`, `ru`, `tr`, `us` - but was not sure is this values can change in the future and if it made sense to have this in the types so opted to `string | undefined`.
Let me know if it would be best to be more specific here - will updated the PR.

Reply:
If you're not sure (especially if the package doesn't crash if you pass unexpected strings), then `string` is fine.